js/index.js

```diff
@@ -0,0 +1,141 @@
+const params = new Proxy(new URLSearchParams(window.location.search), {
```

Comment:
why use Proxy? i don't see any reason to do so... please explain here in a comment or remove it.